### PR TITLE
LPS-93319 DS change-tracking-service

### PR DESCRIPTION
**Pull request by Preston-Crary**

### CODE REVIEW

**Comment by Preston-Crary** [comment]
ci:test:relevant

File: modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/service/impl/CTCollectionLocalServiceImpl.java [hunk 1]
```diff
@@ -107,7 +109,7 @@ public CTCollection deleteCTCollection(CTCollection ctCollection)
 				continue;
 			}
 
-			ctEntryLocalService.deleteCTEntry(ctEntry);
+			ctEntryPersistence.remove(ctEntry);
```
Comment (shuyangzhou) [inline modules/apps/change-tracking/change-tracking-service/src/main/java/com/liferay/change/tracking/service/impl/CTCollectionLocalServiceImpl.java]: @lanceji92674957 @Preston-Crary I don't think this kind of change is safe. The service method has Indexable annotation on it, simply switch to persistence will lose the indexing.

**Comment by Preston-Crary** [comment]
ci:test:relevant

**Comment by shuyangzhou** [comment]
See https://github.com/brianchandotcom/liferay-portal/pull/71514

**Pull request closed without merging**